tests/api-watch/test_diff_module.ml

```diff
@@ -85,7 +85,7 @@ let%expect_test "Modules with both supported and unsupported changes" =
   [%expect
     {|
     Some (Module Main: {Modified (Supported [ Value (x, Removed);
-    Module M: {Modified (Unsupported)}])})|}]
```

Comment:
It might be good to change the code a bit here so that the test output still contains an unsupported change.

Once all of the items are handled this part of the test won't be useful anymore but for now, you should be able to simply replace the `type t` with an exception declaration:
```
type exn += Some_exn
```

None of the issues we opened will cover this so it should fix it for other similar PRs as well.